projects/plugins/boost/app/contracts/Changes_Output.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Contracts;
+
+/**
+ * Modules can implement this interface to indicate that they change the HTML output for the site visitor.
+ */
+interface Changes_Output {}
```

Comment:
Is headers a part of output? What about compression? Is that what this interface asks? Maybe we should use something like `Modifies_HTML` just so that it's specific.

Reply:
Oh 🤔  I didn't realize it includes CSS/JS too.

Yeah. I'm not sure I can come up with a better name....

`Changes_DOM`, `Changes_Page_Output`, `Modifies_Assets_Or_HTML` 🤷  all sound pretty meh to me....

Reply:
I like `Changes_Page_Output`